dojo/tools/twistlock/parser.py

```diff
@@ -1,16 +1,118 @@
+import io
+import csv
 import json
+import hashlib
+
 from dojo.models import Finding
 
 
-class TwistlockParser(object):
-    def __init__(self, json_output, test):
 
-        tree = self.parse_json(json_output)
 
+class TwistlockCSVParser(object):
+
+    @staticmethod
+    def get_field_from_row(row, column):
+        if row[column] is not None:
+            return row[column]
```

Comment:
this if is not required, you can return directly:
```
return row[column]
```
in case row[column] is None, it will return None